fuelup-init.sh

```diff
@@ -1,42 +1,40 @@
 #!/bin/sh
 set -e
-  
+
+FUELUP_DIR=${FUELUP_DIR-"$HOME/.fuelup"}
+
 main() {
-  need_cmd git
-  need_cmd curl
-  need_cmd chmod
-  need_cmd mkdir
-  need_cmd rm
-  need_cmd rmdir
-
-  get_architecture || return 1
-  local _arch="$RETVAL"
-  assert_nz "$_arch" "arch"
-
-  # TODO: update url once we publish releases
-  # releases should support linux/darwin, aarch64 and x86_64, so perhaps in the format:
-  # fuelup-init-x86_64-apple-darwin.tar.gz
-  # fuelup-init-x86_64-unknown-linux-gnu.tar.gz
-  # fuelup-init-aarch64-apple-darwin.tar.gz
-  # fuelup-init-aarch64-unknown-linux-gnu.tar.gz
-  # 
-  # Ideally devs should be able to curl this binary and run it to install forc
-  local _fuelup_url="https://github.com/FuelLabs/fuelup/releases/download/fuelup-init-${_arch}"
-
-  # TODO: testing with just a checked-in binary, remove when we have releases
-  local _tmp_url="https://github.com/FuelLabs/fuelup/raw/binggh/mvp/fuelup/fuelup"
-
-
-  local _dir
-  _dir="$(ensure mktemp -d)"
-  local _file="${_dir}/fuelup"
-
-  ensure downloader "$_tmp_url" "$_file" "$_arch"
-  ensure chmod u+x "$_file"
-
-    if [ ! -x "$_file" ]; then
-        printf '%s\n' "Cannot execute $_file (likely because of mounting /tmp as noexec)." 1>&2
-        printf '%s\n' "Please copy the file to a location where you can execute binaries and run ./rustup-init${_ext}." 1>&2
+    need_cmd git
+    need_cmd curl
+    need_cmd chmod
+    need_cmd mkdir
+    need_cmd rm
+    need_cmd rmdir
+
+    get_architecture || return 1
+    local _arch="$RETVAL"
+    assert_nz "$_arch" "arch"
+
+    mkdir -p "$FUELUP_DIR/bin"
+
+    local _fuelup_version="0.0.2"
```

Comment:
Instead, can you get the latest tagged release? ref: https://github.com/celestiaorg/docs/blob/d73cc3cf571040e9b589b5f194e8fb73f9e7b452/docs/developers/celestia-app.md?plain=1#L18-L20